scripts/cbmc-viewer/cbmc-quiet-xml

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python -u
```

Comment:
There's a general mixing of the shebangs here. Some are indicating that `python3` is expected to run the scripts and some are indicating that `python2` will, but with forward portability.

Is this by design? Why not default to python 3? If there's expectation that python 2 needs to be supported, why are some files asking the environment modifier to default to python 3?

Reply:
These are old scripts used with an old project, and they have been deleted.